doc/api/esm.md

```diff
@@ -474,18 +474,39 @@ See [Loading ECMAScript modules using `require()`][] for details.
 
 ### CommonJS Namespaces
 
+<!-- YAML
+added: v14.13.0
+changes:
+  - version:
+    - REPLACEME
```

Comment:
nit: this is semver-major, it's going to land on a single version, thus having a list is not useful
```suggestion
  - version: REPLACEME
```